src/middlewared/middlewared/etc_files/local/apache24/Includes/webdav.conf

```diff
@@ -25,7 +25,7 @@
 		uid = 0
 		gid = 0
 
-	subprocess.run(['chown', '-Rx', f'{uid}:{gid}', oscmd], check=False)
+	subprocess.run(['chown', '-R', f'{uid}:{gid}', oscmd], check=False)
```

Comment:
Why are we removing x?

Reply:
It meant `stay on one filesystem`, there is no corresponding option in Linux (but there also is no sense in it, we don't mount any filesystems in /etc/apache2)